### DISCUSSION TITLE
Swiper autoplay pause and resume

### DISCUSSION THREAD

**Discussion by chetrix** [nolimits4web/swiper#5619] [Q&A]
How can I  pause/resume Swiperjs Autoplay instead of start/stop which restarts the timer? Is there any workaround wherein on mouse hover it pauses and on leaving it resumes not restarts. I tried pauseOnMouseEnter, but it restarted the autoplay instead of resuming.

I am using custom pagination wherein progress bar is setup. However on mouse hover, slide and bar pauses and on mouse leave , progress resume from the same position and reaches end whereas carousel instead takes delays on changing the slides  Please help.

const swiper = new Swiper(".swiper", {
  modules: [Navigation, Pagination, Autoplay],
  slidesPerView: 1,
  effect: 'fade',
  loop: true,
  autoplay: {
    delay: 3000,
    pauseOnMouseEnter: true,
    disableOnInteraction: false,
  },
  pagination: {
    el: '.swiper-pagination',
    type: 'bullets',
    clickable: 'true',
    renderBullet: function (index, className) {
      return '<span class="' + className + '">' + '<em>'+'</em>' + '<i></i>' + '<b></b>'  + '</span>';
    }
  },
  
  navigation: {
    nextEl: '.swiper-button-next',
    prevEl: '.swiper-button-prev'
  }
});

**Comment by JorenRothman** [comment]
I'm having the same problem it feels like in the current version .stop and .pause are doing the same thing, they both reset the internal timer to 0 however pause should keep it on the current time elapsed.

**Comment by stefanocurnis** [comment]
Same problem here.

**Comment by pedrobernardina** [comment]
:candle: :cake:

**Comment by Flashforge77** [comment]
I have the same problem still in the latest version.

When i pause the autoplay and resume/run it, the timer for the slide is reset to 0 everytime.

**Comment by BidyasagarAnupam** [comment]
In my case `pauseOnMouseEnter: true`, is working properly
